integration_tests/structs_35.py

```diff
@@ -0,0 +1,26 @@
+from lpython import dataclass, field, i32
+from numpy import array
+
+@dataclass
+class X:
+    a: i32 = 123
+    b: bool = True
+    c: list[i32] = field(default_factory=lambda: [1, 2, 3])
+    d: i32[3] = field(default_factory=lambda: array([4, 5, 6]))
+    e: i32 = field(default=-5)
```

Comment:
Is this the same as:
```suggestion
    e: i32 = -5
```
?

Reply:
Yes, I think they are equivalent.

Reply:
If they are equivalent, then I would almost not support the duplicate way of doing it ("just one way of doing things").